crates/bitwarden-crypto/src/sensitive/sensitive.rs

```diff
@@ -67,6 +67,13 @@ impl TryFrom<SensitiveVec> for SensitiveString {
     }
 }
 
+impl From<SensitiveString> for SensitiveVec {
+    fn from(mut s: SensitiveString) -> Self {
+        let value = std::mem::take(&mut s.value);
+        Sensitive::new(Box::new(value.into_bytes()))
```

Comment:
Is this getting rid of `value` after the move?

Reply:
`into_bytes` consumes the String and returns the internal Vec, so there's no data copied or moved.